build_ng2.xml

```diff
@@ -117,6 +117,23 @@
       </sequential>
     </for>
   </target>
+  
+  <target name="cleanupPlugins">
+    <!-- <delete><fileset .../>...</delete> is extremely slow -->
+    <for param="path">
+      <path>
+        <dirset dir="${capstone}" defaultexcludes="false">
+          <include name="**/web/**/*.js"/>
```

Comment:
this must be conditional. you cant remove ".js" unless you see ".js.gz" of the same file name.